modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceDiscountCheckout.testcase

```diff
@@ -0,0 +1,207 @@
+@component-name = "portal-commerce"
+definition {
+
+	property custom.properties = "jsonws.web.service.paths.excludes=";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Shopping Experience";
+
+	setUp {
+		CommerceConfiguration.commerceSetUp(minium="true");
+	}
+
+	tearDown {
+		CommerceConfiguration.commerceTearDown();
+	}
+
+	@description = "This is a test for COMMERCE-10343. Can checkout a product with two discounts, lower discount will be apply"
+	@priority = "3"
+	test CanCheckoutProductWithDiscountLowerDiscountIsApply {
```

Comment:
Can you rename it "CanCheckoutProductWithDiscountsLowerDiscountIsApplied"

Thanks!